docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -922,10 +922,9 @@ Craft Light:
     Category:
         - Magic
     Description: Your skills let you conjure illusory effects.
-    Mechanic: Make a DR 3 Craft Check. On a Success you create an Aura 6 space radiance of light for 15 minutes. On a Failure, the light only lasts for 1 round. You can create a temporary illusory object or infuse an existing object with this property. It can be covered and dimmed. Using a Minor action allows you to extinguish a light. If the light is more than 100 spaces away from you, it disappears
+    Mechanic: Make a DR 3 Craft Check. On a Success you create an Aura 6 space radiance of light for 15 minutes. On a Failure, the light only lasts for 1 round. You can create a temporary illusory object or infuse an existing object with this property. It can be covered and dimmed. Using a Minor action allows you to extinguish a light. If the light is more than 100 spaces away from you, it disappears. Only one instance of Craft Light can be active at any given time. 
```

Comment:
I'd propose that it doesn't require a Craft Check at all to create the light. Speeds up the game by not requiring an additional check, and players don't feel punished by drawing a card to do something simple.